server/pulp/plugins/loader/api.py

```diff
@@ -418,10 +418,12 @@ def load_content_types(types_dir=_TYPES_DIR, dry_run=False, drop_indices=False):
         _logger.critical(msg % {'p': types_dir})
         raise IOError(msg % {'p': types_dir})
     descriptors = _load_type_descriptors(types_dir)
+    definitions = parser.parse(descriptors)
```

Comment:
Could you add some comments here? It looks like these 2 lines are doing the same thing, but I am guessing that the new function works for Mongoengine converted plugins?